tests/test_selected_word_counter.py

```diff
@@ -0,0 +1,143 @@
+import os
```

Comment:
Some nice tests here, although I'm still missing a lot of the smaller (unit) functions getting tested.

Reply:
For now this will be in further PR's